CatchMe/CatchMe/Source/ViewControllers/MainVC.swift

```diff
@@ -204,6 +262,6 @@ extension MainVC: UICollectionViewDelegate {
         pageControl.selectedPage = Int(page)
         
         // Label 내용 변경하는 코드
```

Comment:
요런 자잘한 주석들은 // 보다는 /// 이 더 좋아유! 구분이 되기 땜시롱!
```suggestion
        /// Label 내용 변경하는 코드
```